ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -14,3 +17,27 @@ enum TestError: Int {
 
 extension TestError: ErrorType {
 }
+
+
+internal extension SignalProducerType {
+	/// Halts if an error is emitted in the receiver signal.
+	/// This is useful in tests to be able to just use `startWithNext`
+	/// in cases where we know that an error won't be emitted.
+	func assumeNoErrors() -> SignalProducer<Value, NoError> {
+		return self.lift { $0.assumeNoErrors() }
+	}
+}
+
+internal extension SignalType {
+	/// Halts if an error is emitted in the receiver signal.
+	/// This is useful in tests to be able to just use `startWithNext`
+	/// in cases where we know that an error won't be emitted.
+	func assumeNoErrors() -> Signal<Value, NoError> {
```

Comment:
I have a similar operator in my project that actually accepts the failure message as a parameter:

```swift
foobar.suppressErrors(failingWith: "unexpected error in foobar")
```

The thing I like about this is that it makes the operator harder to use without thinking about the reason why you're suppressing errors. If this method became public, I think that could be a useful affordance.